### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by jschmidb**
This pr adds misc fixes from the opencryptoki 3.21 test.
I may add further commits during the test.

### CODE REVIEW

File: testcases/misc_tests/multi_instance.c [hunk 1]
```diff
@@ -183,8 +183,8 @@ int main(int argc, char **argv)
     // generate a key pair for slot 1
     testcase_new_assertion();
     rv = do_GenerateTokenRSAKeyPair(slot_id1, session1,
-                                    (CK_BYTE *)"RSA-1024-SLOT1",
-                                    1024, &hPubKey1, &hPrivKey1);
+                                    (CK_BYTE *)"RSA-2048-SLOT1",
+                                    2048, &hPubKey1, &hPrivKey1);
```
Comment (ifranzki) [inline testcases/misc_tests/multi_instance.c:187]: The same should be done for the fork test, I guess.

Comment (jschmidb) [reply testcases/misc_tests/multi_instance.c:187]: Yep, right ... will do.

Comment (jschmidb) [reply testcases/misc_tests/multi_instance.c:187]: BTW: in both cases we can leave the pub_exp as it is (0x10001). Smaller exponents would not be allowed in fips mode.

File: testcases/pkcs11/getobjectsize.c [hunk 2]
```diff
@@ -93,7 +95,7 @@ CK_RV do_GetObjectSize(void)
         return rc;
     }
 
-    printf("C_GetObjectSize test passed\n");
+    testcase_pass("C_GetObjectSize test passed\n");
```
Comment (jschmidb) [inline testcases/pkcs11/getobjectsize.c:98]: just saw that I can remove the newline in testcase_pass, there are enough newline in the macro ... will check all places.

**Comment by jschmidb** [comment]
... and saw the travis errors, will push again in a minute ...

**Comment by ifranzki** [comment]
still travis errors

**Comment by jschmidb** [comment]
Yes, saw it ... actually I think we can remove the msg insert as the msg itself says what happened. Re-pushed ...

**Review by ifranzki — APPROVED** [review]
lgtm

**Comment by jschmidb** [comment]
> Travis looks good now. For whatever reasons the s390x virtual machines don't boot in travis at the moment. Lets see if they completed by tomorrow.

Yep, I have the same problem with libica ... let's wait

**Review by vkanjur — APPROVED** [review]

**Pull request merged**